### PR TITLE
RHPAM-3510: upgrade minor versions

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[RHPAM-3510](https://issues.redhat.com/browse/RHPAM-3510)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest</b> (using <i>this</i> e.g. <b>Jenkins retest this</b> optional but no longer required)
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins run fdb

**Comment by mbiarnes** [comment]
@mareknovotny The fdb failed - the other #1735 fdb didn't fail, at least not in appformer. There is one of the upgrades that is problematic here.

File: pom.xml [hunk 1]
```diff
@@ -151,7 +151,7 @@
     <version.org.apache.maven.wagon>3.0.0</version.org.apache.maven.wagon>
     <version.org.apache.neethi>3.1.1</version.org.apache.neethi>
     <version.org.apache.poi>4.1.2</version.org.apache.poi>
-    <version.org.apache.sshd>2.3.0</version.org.apache.sshd>
+    <version.org.apache.sshd>2.4.0</version.org.apache.sshd>
```
[the remark this answers is not in the document]
Comment (mareknovotny) [reply pom.xml:154]:
```suggestion
    <version.org.apache.sshd>2.3.0</version.org.apache.sshd>
```

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mbiarnes** [comment]
Linux run fdb

**Comment by mbiarnes** [comment]
Jenkins run fdb

**Comment by mbiarnes** [comment]
Jenkins run fdb

**Comment by mbiarnes** [comment]
Jenkins run fdb

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mareknovotny) [reply pom.xml:154]:
```suggestion
    <version.org.apache.sshd>2.3.0</version.org.apache.sshd>
```

Comment (mareknovotny) [inline pom.xml:154]:
```suggestion
    <version.org.apache.sshd>2.3.0</version.org.apache.sshd>
```

**Comment by mareknovotny** [comment]
Jenkins run fdb

**Comment by mareknovotny** [comment]
Failures in JBPM:
- org.jbpm.test.container.test.WebServiceInvokeTest.testWebServiceSync
- org.jbpm.test.container.test.WebServiceInvokeTest.testMultipleImports
- org.jbpm.test.container.test.WebServiceInvokeTest.testInterfaceNamePassedViaAssignemnt
- org.jbpm.test.container.test.WebServiceInvokeTest.testWebServiceOneWay
- org.jbpm.test.container.test.WebServiceInvokeTest.testWebServiceAsync
- org.jbpm.remote.ejb.test.task.EWebServiceTaskTest.testOneWayWebServiceWorkItem
- org.jbpm.remote.ejb.test.task.EWebServiceTaskTest.testSyncWebServiceWorkItem
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testServiceInvocationWithMultipleIntParams
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testServiceInvocationProcessWith2WSImports
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testOneWayServiceInvocationProcessWSHandler
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testServiceInvocationWithMultipleParams
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testTimeoutViaParameters
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testServiceInvocationProcessWith2WSImportsWSHandler
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testTimeoutViaSystemProperty
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testOneWayServiceInvocation
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testAsyncServiceInvocation
- org.jbpm.process.workitem.bpmn2.JaxWSServiceTaskTest.testServiceInvocation

File: pom.xml [hunk 2]
```diff
@@ -183,7 +183,7 @@
     <version.org.eclipse.sisu>0.3.2</version.org.eclipse.sisu>
     <version.org.eclipse.emf.gwt>2.9.0</version.org.eclipse.emf.gwt>
     <version.org.glassfish>3.1.2</version.org.glassfish>
-    <version.org.glassfish.jaxb>2.3.2</version.org.glassfish.jaxb>
+    <version.org.glassfish.jaxb>2.3.3-b02</version.org.glassfish.jaxb>
```
Comment (mareknovotny) [inline pom.xml:186]:
```suggestion
    <version.org.glassfish.jaxb>2.3.2</version.org.glassfish.jaxb>
```

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
Failed 3 tests:

- org.uberfire.ext.metadata.backend.infinispan.provider.InfinispanPingServiceTest.testPingSuccess
- org.kie.karaf.itest.blueprint.KieBlueprintjBPMPersistenceKarafIntegrationTest.testStartProcessEnv
- org.jbpm.executor.impl.jms.JmsAvaiableJobExecutorTest.testAsyncAuditErrorProducer

I think they are not related and are random failures due environment

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**